fast/stages/0-bootstrap/checklist.tf

```
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  # group mapping from checklist to ours
+  _cl_groups = {
+    BILLING_ADMINS = local.groups.gcp-billing-admins
```

```suggestion
    BILLING_ADMINS = local.groups_iam.gcp-billing-admins 
```
Using groups_iam here simplifies `local.checklist.group_iam` creation - you won't need to check if entry is a group

---

nope, this will mess up the keys for additive bindings...